File: far_plugin/app/src/main/java/com/westryank/farplugin/OutputResultTypeEnum.java
package com.westryank.farplugin;

public enum OutputResultTypeEnum {
    Final(0),
    SegmentationMask(1),
    NoFineTuning(2);

    public final int value;

    private OutputResultTypeEnum(int inValue) {
        value = inValue;
    }

    public static OutputResultTypeEnum FromInt(int inValue) {
        return OutputResultTypeEnum.values()[inValue];
    }
}

File: far_app/Assets/Scripts/Settings/OutputResultTypeEnum.cs
using System.Collections;
using System.Collections.Generic;

// If you change these values, also update them in far_plugin project.
public enum OutputResultTypeEnum
{
    [EnumName("Final")]
    final = 0,
    [EnumName("Segmentation Mask")]
    segmentationMask = 1,
    [EnumName("No Fine Tuning")]
    noFineTuning = 2
}